### PR TITLE
cd: add Portuguese translation

### PR DESCRIPTION
**Pull request by FilipaDurao**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Feel free to open 1 big PR if you're translating lots of pages, with 1 commit per page :-)

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_PT/common/cd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# cd
+
+> Mudar o diretório atual.
```
Comment (schneiderl) [inline pages.pt_PT/common/cd.md:3]: Perhaps we should add the link to the official docs here (https://docs.microsoft.com/windows-server/administration/windows-commands/cd.)

**Review by schneiderl — APPROVED** [review]
Looks good. :)

**Review by schneiderl — CHANGES_REQUESTED** [review]
Did not meant to approve.

File: pages.pt_PT/common/cd.md (hunk 1 above)
Comment (sbrl) [reply pages.pt_PT/common/cd.md:3]: This is the `cd` command in the `common` directory. `cd` functions differently on different platforms, so I'm not sure it's suitable to link to the Windows documentation here.

Comment (FilipaDurao) [reply pages.pt_PT/common/cd.md:3]: @schneiderl Should I do anything here or is the problem solved? Should I add the link or not?

Comment (schneiderl) [reply pages.pt_PT/common/cd.md:3]: @FilipaDurao do not add the link. @sbrl comment is on point, we should not add something system specific to the common directory.

**Review by schneiderl — APPROVED** [review]

**Pull request merged**